github/github_test.go

```diff
@@ -911,3 +911,41 @@ func TestBasicAuthTransport_transport(t *testing.T) {
 		t.Errorf("Expected custom transport to be used.")
 	}
 }
+
+func TestFormatRateLimitResetDuration(t *testing.T) {
+	d := 120*time.Minute + 12*time.Second
+	got := formatRateLimitResetDuration(d)
+	want := "[rate reset in 120m12s]"
+
+	if got != want {
+		t.Errorf("Format is wrong. want: %v, got: %v", want, got)
```

Comment:
To be consistent with the most prevalent style, the order should always be "got", then "want". That makes test output more readable. So make this:

```Go
t.Errorf("Format is wrong. got: %v, want: %v", got, want)
```

And in other lines as well.